docs/azure/vs-azure-tools-debug-cloud-services-virtual-machines.md

```diff
@@ -102,27 +102,27 @@ You can debug programs that run on Azure virtual machines by using Server Explor
 
 ### To debug an Azure virtual machine
 
-1. In Server Explorer, expand the Virtual Machines node and select the node of the virtual machine that you want to debug.
+1. In **Server Explorer**, expand the Virtual Machines node and select the node of the virtual machine that you want to debug.
```

Comment:
Keep original.

Reply:
Thank you! :) I'll keep the title-style capitalization only.